MiniTeks/mks-server
# Triggers

added  triggers
* added tekton resources for clone-apply pipeline
* added triggers to auto deploy on a new commit to the 
* added triggers to run tests
* added run.sh to prepare the cluster for mks-server

### How to test
1. Setup the OC cluster and Install OSPipelines
2. Run the script to apply dependencies ```./scripts/run.sh```
3. To test pipelineruns-
              i. create the pipelineruns resources using ```oc create -f <pipelinerun>```
              ii. check the logs using ```tkn pipelinerun logs -f```
4. To expose the trigger services-
              i. ``` oc expose svc el-<trigger-service-name>```
              ii. Add url to github webhook - 
                   ```echo "$(oc  get route el-<trigger-service-name> --template='http://{{.spec.host}}')"```
